onos-config/templates/deployment.yaml

```diff
@@ -47,6 +47,14 @@ spec:
           env:
             # *_NAMESPACE and *_NAME environment variables are recognized by onos-lib-go utilities.
             # These variables should always be defined.
+            - name: GRPC_GO_LOG_VERBOSITY_LEVEL
```

Comment:
what loggers are these? log levels for onos-config and onos-topo can be controlled from the values in the helm chart.

Reply:
I would suggest to remove them to avoid confusion with our loggers.

Reply:
added the logs under flag.